tests/js/setup.ts

```diff
@@ -189,6 +189,21 @@ const routerFixtures = {
     ...params,
   }),
 
+  routeComponentProps: (
```

Comment:
overall in favor of this since `initializeOrg` is stupid

Reply:
Yeah, initializeOrg is doing waaay too much